js/src/lib/Details/index.js

```diff
@@ -118,17 +119,37 @@ class Details extends Component {
       }
 
       this.getGithub({
-        url: `repos/${this.state.githubRepo.user}/${this.state.githubRepo
-          .project}/stats/commit_activity`,
+        url: `repos/${this.state.githubRepo.user}/${
+          this.state.githubRepo.project
+        }/stats/commit_activity`,
         state: 'activity',
       });
 
       this.getGithub({
-        url: `repos/${this.state.githubRepo.user}/${this.state.githubRepo
-          .project}`,
+        url: `repos/${this.state.githubRepo.user}/${
+          this.state.githubRepo.project
+        }`,
         state: 'github',
       });
     }
+
+    const { name, version } = this.state;
+
+    get({
+      url: `https://bundlephobia.com/api/size?package=${name}@${version}`,
+      type: 'json',
+      headers: {
+        'User-Agent': `yarn website; ${navigator.userAgent}`,
```

Comment:
Does this actually work? I thought User-Agent was a "protected" header that could not be overwritten.

Reply:
Ahh, never mind, TIL this changed at some point:

> Note: The User-Agent header is no longer forbidden, as per spec — see forbidden header name list (this was implemented in Firefox 43), so it can now be set in a Fetch Headers object, via XHR setRequestHeader(), etc.

(from https://developer.mozilla.org/en-US/docs/Glossary/Forbidden_header_name)

Reply:
Yeah, it worked in my devtools

Reply:
@Haroenv Is it an ok practice to do so? I would have created a dedicated "X-header" but WFM if that's completely OK.

Reply:
Changed it to `X-Bundlephobia-User`, cc @pastelsky